nano/node/node.cpp

```diff
@@ -1270,15 +1270,17 @@ void nano::node::add_initial_peers ()
 	}
 }
 
-void nano::node::block_confirm (std::shared_ptr<nano::block> const & block_a)
+std::shared_ptr<nano::election> nano::node::block_confirm (std::shared_ptr<nano::block> const & block_a)
 {
 	scheduler.manual (block_a);
 	scheduler.flush ();
```

Comment:
git flush is broken and the plan was to stop using it rather than fix it.
Therefore any code that depends on it is also broken.